protos/validator.proto

```diff
@@ -33,59 +33,63 @@ message Message {
         TP_REGISTER_REQUEST = 1;
         // Registration response from the validator to the transaction processor
         TP_REGISTER_RESPONSE = 2;
+        // Tell the validator that the transaction processor won't take any more transactions
```

Comment:
I could add a break of 10 (20?) numbers between top level namespace message types. So TP would end at 12, but CLIENT wouldn't start until 23 (33?).